playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -623,6 +623,23 @@ EDXAPP_COMPREHENSIVE_THEME_SOURCE_REPO: ""
 # EDXAPP_COMPREHENSIVE_THEME_SOURCE_REPO
 EDXAPP_COMPREHENSIVE_THEME_VERSION: ""
 
+# supply a default site theme keyed to the main LMS domain
+# only works with EDXAPP_ENABLE_COMPREHENSIVE_THEMING set to true, 
+# otherwise, ignored - 
+# have to use this way of defining dict in string format
+# so the variable is interpolated in the key, but if not 
+# using variable key names a more standard format can be used
+EDXAPP_SITE_THEMES: "{
```

Comment:
In order to not have the key end up as `"{# EDXAPP_LMS_SITE_THEME #}"` here, I had to use this string-ish way of defining the dict.  Right now I'm not using the key for anything really, but I wanted to use the LMS site domain as they key here in case we want to make other Ansible tasks for setting up Django `Site`s and connected `Site Themes` from this value.